simulation-model/simulation-model_instance.json

```diff
@@ -0,0 +1,194 @@
+{
+    "@context": {
+        "general": "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/",
+        "simulation-model": "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/simulation-model/",
+        "sh": "http://www.w3.org/ns/shacl#",
+        "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "skos": "http://www.w3.org/2004/02/skos/core#",
+        "georeference": "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/georeference/"
+    },
+    "@id": "did:web:registry.gaia-x.eu:SimulationModel:2Jt36BA9ODxKZJ5yWRtF4BLDUkuvFL59IPra",
+    "@type": "simulation-model:SimulationModel",
+    "simulation-model:general": {
+        "@type": "general:General",
+        "general:description": {
+            "@type": "general:Description",
+            "general:name": {
+                "@value": "Simulation Model Example",
+                "@type": "xsd:string"
+            },
+            "general:description": {
+                "@value": "A sample description of a none existing simulation model.",
+                "@type": "xsd:string"
+            }
+        },
+        "general:data": {
+            "@type": "general:Data",
+            "general:size": {
+                "@value": "1.0",
+                "@type": "xsd:float"
+            },
+            "general:contractId": {
+                "@value": "1234567890",
+                "@type": "xsd:string"
+            },
+            "general:recordingTime": {
+                "@value": "2024-06-26T09:00:00",
+                "@type": "xsd:dateTime"
+            }
+        },
+        "general:links": {
+            "@type": "general:Links",
+            "general:data": {
+                "@type": "general:Link",
+                "general:url": {
+                    "@value": "https://github.com/openMSL/sl-1-1-reflection-based-radar-object-model",
+                    "@type": "xsd:anyURI"
+                },
+                "general:type": "Asset"
+            },
+            "general:media": {
+                "@type": "general:Link",
+                "general:url": {
+                    "@value": "https://inavlid.link.com/media",
+                    "@type": "xsd:anyURI"
+                },
+                "general:type": "Video"
+            }
+        },
+        "general:bundleData": {
```

Comment:
This is not part of general anymore.

Reply:
Yes. Somehow schema checker did not complained. Removed this.